build-system/global-configs/prod-config.json

```diff
@@ -15,6 +15,7 @@
   "disable-a4a-non-sd": 1,
   "amp-cid-backup": 1,
   "story-ad-placements": 0.01,
+  "story-ad-page-outlink": 0,
```

Comment:
Rever the two changes in `canary-config.json` and `prod-config.json`. We're preferring `client-side-experiments-config.json`. See https://github.com/ampproject/amphtml/tree/main/build-system/global-configs#client-side-experiments-configjson

**Note**: The change to `client-side-experiments-config.json` should be done in a separate PR. Changes to the file are propagated to traffic immediately.

Reply:
Sounds good, the changes were reverted and I created a new PR Here:
https://github.com/ampproject/amphtml/pull/36030